Documentation/api-reference.md

```diff
@@ -82,25 +114,40 @@ public static void registerExtension()
 
 ##### Example
 ```java
-import com.adobe.marketing.mobile.Edge
+import com.adobe.marketing.mobile.Edge;
```

Comment:
Let's remove the imports from the sample. We have that in getting started.

Reply:
The documentation for the other extensions do include the import statement just for the `registerExtension` API, so I think we can leave it here as well.